DMX.Core.Api/Brokers/ReverbApis/ReverbApiBroker.Devices.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System.Threading.Tasks;
+using DMX.Core.Api.Models.External.ExternalLabs;
+
+namespace DMX.Core.Api.Brokers.ReverbApis
+{
+    public partial class ReverbApiBroker
+    {
+        private const string ReverbServiceType = "AzureIotHub";
+
+        public async ValueTask<ExternalLabsCollection> GetAvailableDevicesAsync(string reverbServiceId)
+        {
+            const string RelativeUrl = "api/GetAvailableDevicesAsync";
+
+            ExternalLabsServiceInformation ReverbServiceProperties = new ExternalLabsServiceInformation
+            {
+                ServiceType = ReverbServiceType,
+                ServiceId = reverbServiceId
+            };
+
+            return await this.PostAync<ExternalLabsServiceInformation, ExternalLabsCollection>($"{RelativeUrl}?code={this.accessKey}", ReverbServiceProperties);
```

Comment:
We need to break this into < 120 chars line.
Also use named parameters when the input doesn't match in part or in full the parameter name.
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Methods.md#104-passing-parameters